x-pack/plugins/index_management/public/application/components/component_templates/__jest__/client_integration/component_template_edit.test.tsx

```diff
@@ -120,6 +120,60 @@ describe('<ComponentTemplateEdit />', () => {
           }),
         })
       );
+      // Mapping rollout modal should not be opened
```

Comment:
Thanks for adding comments! For comments like these, would you mind adding some more information about _why_ we want the UI to behave this way? In other words, what's the intention behind this decision not to open the mapping rollout modal?

Reply:
Just updated the comment, you are right it make a lot more sense to add the "why"